src/discv5.rs

```diff
@@ -479,6 +488,93 @@ impl Discv5 {
         }
     }
 
+    // Use find_topic to find the Enrs the shortest XOR distance from the topic hash,
+    // and send the topic query to these nodes
+    pub fn topic_query_req(
+        &self,
+        enr: Enr,
```

Comment:
This ENR is supposed to be our ENR right?
We send our ENR to other nodes so they know to register it, correct?
If this is the case, I dont think we need to expose this as a parameter here. The service knows about our local enr, so could use just use it there without us having to pass it in. 
I saw that in responses, we reject ENR's that don't match the source, so putting someone else's here would just get rejected right?

Reply:
No the idea for this parameter is that the request is sent to a specific node. The local enr is added to the request on the Service layer. Here is some WIP code I used for testing, of a fork of yours :) https://github.com/emhane/discv5-cli/blob/2318f77f4017b2f9f35b5f91be11628c8e66c462/src/topic/mod.rs#L127